snapcraft/how-to/qt-applications/qt-applications.md

```diff
@@ -1,265 +1,486 @@
-## Why are snaps good for Qt5 and KDE Frameworks applications?
+Snapcraft can be used to package and distribute Qt 5 applications, including applications that use the KDE Frameworks 5 add-on libraries, in a way that enables convenient installation by users.
 
-Snapcraft bundles necessary libraries required by the application, and can configure the environment for confinement of applications for end user peace of mind. Developers can ensure their application is delivered pre-packaged with libraries which will not be replaced or superseded by a distribution vendor.
+The process of creating a snap for a Qt 5 and KDE Frameworks 5 application builds on standard tools like `cmake`, `make` and `qmake`. This enables developers to re-use the application's existing build system when creating a snap, making packaging easy.
 
-Here are some snap advantages that will benefit many Qt applications:
+[note]
 
-* **Snaps are easy to discover and install**
-  Millions of users can browse and install snaps graphically in the Ubuntu Software Center, the Snap Store or from the command-line.
-* **Snaps install and run the same across Linux**
-  They bundle the latest version of Qt5 and KDE Frameworks, along with all of your app's dependencies, be they binaries or system libraries.
-* **You control the release schedule**
-  You decide when a new version of your application is released without having to wait for distributions to catch up.
-* **Snaps automatically update to the latest version**
-  Four times a day, users' systems will check for new versions and upgrade in the background.
-* **Upgrades are safe**
-  If your app fails to upgrade, users automatically roll back to the previous revision.
```

Comment:
I took out this section out as it seemed misplaced for a how-to, and the only other platform how-to guides it features in are the ones for GTK applications. The top section (lines 1-2) is based on a similar section used in other guides (e.g. the C/C++ how-to).